src/middlewared/middlewared/plugins/disk.py

```diff
@@ -468,15 +476,8 @@ def disk_extend(self, disk):
         async for i in await self.middleware.call('boot.get_disks'):
```

Comment:
How about an `async def __get_reserved(self)`, this code repeats the one above (at lines 54..55)